source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -138,6 +140,19 @@ Settings
        | **Default**: ``false``
        | **Accepted Values**: ``true`` or ``false``
 
+   * - | **change.stream.document.key.as.key**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Whether to use the document key for the source record key if
+         the document key is present. When this setting is ``true``,
```

Comment:
Suggestion:
I think it could be good to use similar spacing and phrasing as "publish.full.document.only" for consistency and precision.

I think "when set to true" since that more precisely describes its value than "when this setting is true".